src/core/link.js

```diff
@@ -205,7 +205,9 @@
          * @return {String} The URI updated with the protocol.
          */
         _getCompleteURI: function(URI) {
-            if (!REGEX_URI_SCHEME.test(URI)) {
+            if (URI.indexOf('@') && URI.match(/[^\/]+@/)) {
```

Comment:
This regex should be exposed as an variable in the same way `REGEX_URI_SCHEME` is.